girder/api/v1/system.py

```diff
@@ -108,7 +108,18 @@ def setSetting(self, params):
         .errorResponse('You are not a system administrator.', 403)
         .errorResponse('Failed to set system setting.', 500))
 
-    @access.admin
+    @access.admin(scope=TokenScope.SETTINGS_READ)
```

Comment:
That's correct. 

> However, those tokens still have to satisfy the admin user requirement, so I don't totally understand what their use case would be.

The case here would be an admin user granting a third-party app the right to read system settings. That third-party app would then have permission that is normally reserved for admin user, but not *all* permissions granted to admins, just the ability to read system settings. This is why it's easiest (though not strictly required) to try and divide endpoints cleanly (read: unconditionally) into scopes.